src/app/multistepform/multistepform.component.css

```diff
@@ -0,0 +1,103 @@
+.form__container {
+  background-color: #ffffff;
+  box-shadow: 8px 9px 20px #e1e8f1;
+  display: flex;
+  flex-direction: column;
+  height: 80%;
+}
+.form__leftSide {
```

Comment:
Please for your naming kindly use a descriptive name instead of the position of elements for naming them. in your case the component is a Stepper component so the className can be something like stepperContainer or something descriptive instead of form__leftSide.